src/cli.ts

```diff
@@ -28,6 +28,12 @@ const options = {
 		default: process.env["TABLE_PROPS"] ?? "allowApiCrud: true",
 		description: `Example only authenticated, set: "allowApiCrud: (r) => r?.authenticated() ?? false"`,
 	},
+	"with-enums": {
+		default: process.env["WITH_ENUMS"]
+			? process.env["WITH_ENUMS"] === "true"
+			: true,
```

Comment:
```suggestion
	"with-enums": {
		default: process.env["WITH_ENUMS"] === "true"

```

Reply:
So if `process.env["WITH_ENUMS"]` is not set, it will not generate enums?
By default, it's good to have enums.

Maybe the flag could be: "SKIP_ENUM" to revert the logic?